### DISCUSSION TITLE
Accessing Group

### DISCUSSION THREAD

**Discussion by kb12abz** [sergiodxa/remix-auth#274] [Q&A]
I am looking to get more user details. I am getting basic user profile data back such as name, displayName, email etc. But I cant seem to find how to access the groups the user has access to. Does this package support this?

cheers

**Comment by sergiodxa** [comment]
I suppose you talk about the OAuth2Strategy, Remix Auth works with any strategies, not only OAuth2.

The OAuth2Strategy has a normalized user profile, but sub strategies can add extra fields, the Auth0Strategy for example adds a `__json` field with all the information that Auth0 gives you.

And finally, since you have the access token, you can fetch any other data you need before returning in the strategy callback.
